commands/fork.go

```diff
@@ -112,16 +112,15 @@ func fork(cmd *Command, args *Args) {
 		originURL := originRemote.URL.String()
 		url := forkProject.GitURL("", "", true)
 
-		// Check to see if the remote already exists and points to the fork.
```

Comment:
What's the point of this check? Even if you have a remote called the same way of `newRemoteName` but pointing to a different project, `git` will later block you

Reply:
Not sure why this check is here, but I believe it was wanted behavior. Is it hard to preserve?

Reply:
no, I've left split the change in two commits. Only applying the second will remove that behaviour.